thorlcr/activities/null/thnullslave.cpp

```diff
@@ -75,7 +75,7 @@ class CNullSlaveActivity : public CSlaveActivity, public CThorDataLink
 
     virtual bool isGrouped()
     {
-        return queryHelper()->queryOutputMeta()->isGrouped();
+        return false;
```

Comment:
Is this always true?  What happens if a grouped dataset is filtered so it contains no records, and that is an input to an IF())?